src/Symfony/Component/HttpKernel/MicroKernel.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel;
+
+use Symfony\Bundle\FrameworkBundle\Routing\RouteCollectionBuilder;
+use Symfony\Component\Config\Loader\LoaderInterface;
+use Symfony\Component\Config\Resource\FileResource;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * A Kernel that provides configuration hooks.
+ *
+ * @author Ryan Weaver <ryan@knpuniversity.com>
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+abstract class MicroKernel extends Kernel
```

Comment:
The problem with "stand-alone" is that it's spelled with a dash, which will make people think: should I spell it as `StandaloneKernel` or `StandAloneKernel`. Besides, the dictionary describes "stand-alone" as: *"able to operate without control from another system"*. Is that entirely true in this case?